napari/utils/_tests/test_notification_manager.py

```diff
@@ -138,10 +143,13 @@ def _raise():
         exception_thread.start()
         time.sleep(0.02)
 
-        if PY38_OR_HIGHER:
-            threading.excepthook(sys.exc_info())
-        else:
-            sys.excepthook(*sys.exc_info())
+        try:
+            raise ValueError("a")
+        except ValueError:
+            if PY38_OR_HIGHER:
+                threading.excepthook(sys.exc_info())
+            else:
+                sys.excepthook(*sys.exc_info())
```

Comment:
We no longer support 3.7, so I guess this can be dropped?

Reply:
done